storybookjs/storybook
# Vue: Update events binding in Vue render

Issue: https://github.com/storybookjs/storybook/issues/19318

## What I did

- Vue2 renderer renders only `v-bind="$props"` in component template which missed the right way of binding custom events in Vue 2 with `v-on` or `@...`, in this PR, we base on `argType.table.category` `events` to add events binding into template.

## How to test

```bash
cd code
yarn task --task dev --template vue-cli/vue2-default-js 
open http://localhost:6006/?path=/story/lib-store-hooks--use-state
```

Open `Interactions` panel and verify all tests are passed

<img width="1499" alt="image" src="https://user-images.githubusercontent.com/51897872/202253476-f04e70a0-94b9-4b34-aaea-860f7b36cdea.png">
